man/systemd.exec.xml

```diff
@@ -936,7 +936,18 @@
         <para>Note that for each unit making use of this option a PAM session handler process will be maintained as
         part of the unit and stays around as long as the unit is active, to ensure that appropriate actions can be
         taken when the unit and hence the PAM session terminates. This process is named <literal>(sd-pam)</literal> and
-        is an immediate child process of the unit's main process.</para></listitem>
+        is an immediate child process of the unit's main process.</para>
+
+        <para>Note that when this option is used for a unit it is very likely (depending on PAM configuration) that the
+        main unit process will be migrated to its own session scope unit when it is activated. This process will hence
```

Comment:
hmm? not following? i try to avoid just writing "scope" when I refer to a unit of type scope, and here I really do. The english word "scope" is used in many other contexts too, and I have the suspicion that many users are not aware of the "scope unit" concept in systemd, hence I try to be precise here, to clarify this is about units.

(this is different for the term "service" btw. Sevices are the most relevant of the unit types systemd manages, and if users understood something about systemd than probably that "unit" is the concept systemd uses to abstract the "service" concept...)

Reply:
Ah, sure. In that case it should stay as it is.